client/tinychain/graph/app.py

```diff
@@ -209,3 +225,20 @@ def upsert(self, key, values):
             return After(Table.upsert(self, key, values), updates)
 
     return GraphTable(table_schema)
+
+
+def create_schema(schemas: list[TableSchema]) -> Schema:
+    """Create a graph schema using model schemas."""
+    graph_schema = Schema()
+    indices = {}
+
+    for s in schemas:
+        name = s.key[0].name.removesuffix("_id")
```

Comment:
[nit] it's clearer to add an `assert` when making assumptions like this about the structure of the input data